code/game/objects/items/storage/pouch.dm

```diff
@@ -372,10 +372,37 @@
 	storage_slots = 2
 	icon_state = "radio"
 	draw_mode = 1
-	desc = "It can contain two handheld radios."
+	desc = "It can contain two handheld radios"
 	can_hold = list("/obj/item/device/radio")
 
 
+/obj/item/storage/pouch/field_pouch
```

Comment:
All of these fit in the large general pouch and webbing.

Reply:
You can make it fit there then, or alter its size.

Reply:
Or we can add a pouch to essentials that people would be more inclined to take and use as it holds and comes preloaded with important and useful gear, and is part of that crucial tone setting kit, thus encouraging people to actually do the job they're intended to instead of taking a flamer/HPR and being a grunt +1.